uxtesting.md

```diff
@@ -1,44 +0,0 @@
-# UX Testing: Sample of responses
```

Comment:
While valuable, the `uxtesting.md` file is quite old now, and I don't think it's appropriate to maintain in the repo. We could move its contents into the repo wiki, or some other place, if we wanted to.